### ISSUE TITLE
GENERAL FEEDBACK

### ISSUE THREAD

**Issue by everitt-andrew** [leonardoz15/Portfolio-grayscale#4]
I thought that your website looks very good. The theme that you chose is clean and the images and text you've included thus far works well. Is the image used at the top under the Creative Commons license?

**Comment by leonardoz15** [comment]
Yes, as I have noted in the README all images are used under public domain from flickr creative commons.
